hack/chaos-result.j2

```diff
@@ -2,11 +2,14 @@
 apiVersion: litmuschaos.io/v1alpha1
 kind: ChaosResult
 metadata:
-  # name of the litmus testcase
-  name: {{ c_experiment }} 
+  {% if instance_id is defined %}
+  name: {{ c_experiment }}{{ instance_id }}
```

Comment:
In our naming convention , we append two name by "-" .  Why we don't put  "-" between c_experiment and instance_id.?

Reply:
Thank you, added!